honeybee_vtk/legend_parameter.py

```diff
@@ -0,0 +1,525 @@
+                legend. Defaults to (0, 100).
+            show_legend: A boolean to set the visibility of a legend in Scene. If not
+                set, it will be set to False. Defaults to None.
+            orientation: An Orientation object that sets the orientation of the legend in
+                the scene. If not set, the legend will be oriented vertically.
+                Defaults to None.
+            position: A tuple of two decimal values. The values represent the percentage
+                of viewport width and the percentage of viewport height. These
+                percentages in decimal numbers will define the location of the start
+                point of the legend. For example, a value of (0.0, 0.5) will place the
+                start point of the legend at the left most side of the legend and at the
+                50% height of the viewport. If not specified (0.5, 0.0) will be used.
+                Default to None.
+            width: A decimal number representing the percentage of viewport width that
+                will be used to define the width of the legend. A value of 0.45 will
+                make the width of legend equal to the 45% width of the viewport. If not
+                set, the value of 0.45 will be used. Defaults to None.
+            height: A decimal number representing the percentage of viewport height that
+                will be used to define the height of the legend. A value of 0.05 will
+                make the height of legend equal to the 5% height of the viewport. If not
+                set, the value of 0.05 will be used. Defaults to None.
+            number_of_colors: An integer representing the number of colors in a legend.
+                Defaults to None.
+            number_of_labels: An integer representing the number of text labels on a
+                legend. Default to None.
+            label_type: A LabelType object.
+            label_position: 0 or 1. The value of 0 would mean that the labels and the
+                title would not precede the legend. The value of 1 would mean that the
+                labels and the title would precede the legend.. If not set, the labels
+                and the title will not precede the legend. Defaults to None.
+            label_font: A Font object. Defaults to None.
+            title_font: A font object. Defaults to None.
+        """
+
+    def __init__(
+            self,
+            name: str = None,
+            colors: Colors = None,
+            range: Tuple[int, int] = None,
+            show_legend: bool = None,
+            orientation: Orientation = None,
+            position: Tuple[float, float] = None,
+            width: float = None,
+            height: float = None,
+            number_of_colors: int = None,
+            number_of_labels: int = None,
+            label_type: LabelType = None,
+            label_position: int = None,
+            label_font=None,
+            title_font=None) -> None:
```

Comment:
Can we put a better default than None? You have default values for each of these arguments. Why don't you just expose them?

Reply:
Complied.